### PR TITLE
prefetch google analytics and github - fix #340

### LINKED ISSUE #340: Put DNS lookup for analytics and github into head
screenshot 2014-12-22 14 14 24

### PR DESCRIPTION
**Pull request by marcobiedermann**

### CODE REVIEW

File: templates/partials/head.tpl [hunk 1]
```diff
@@ -10,6 +11,9 @@
   <meta name="description" value="Perf Tooling Today lists a lot of tools that can be used to make the web faster, because performance matters!">
   <meta name="apple-mobile-web-app-title" content="perf-tooling">
 
+  <link rel="dns-prefetch" href="http://www.google-analytics.com/">
+  <link rel="dns-prefetch" href="http://www.https://avatars.githubusercontent.com/">
+
```
Comment (stefanjudis) [inline templates/partials/head.tpl]: @marcobiedermann ping.

**Comment by marcobiedermann** [comment]
What is the correct way how to do this? Do you have any resources? Or are you going to fix it?

**Comment by stefanjudis** [comment]
@marcobiedermann In genrral it looks fine but `http://www.https://avatars.gi` looks like a typo?

**Comment by stefanjudis** [comment]
:+1: :)

**Pull request merged**